dnf-behave-tests/dnf/environment.py

```diff
@@ -247,19 +282,28 @@ def before_scenario(context, scenario):
         return
 
     # if "dnf5" is in the commandline tags, turn on dnf5 mode
+    # if "dnfdaemon" turn on dnfdamon mode and dnf5 mode
     dnf5_mode = False
+    dnfdaemon_mode = False
     for ors in context.config.tags.ands:
         if "dnf5" in ors:
             dnf5_mode = True
-            break
+        if "dnfdaemon" in ors:
+            dnf5_mode = True
+            dnfdaemon_mode = True
 
     context.dnf = DNFContext(context.config.userdata,
                              force_installroot='force_installroot' in scenario.tags,
                              no_installroot='no_installroot' in scenario.effective_tags,
-                             dnf5_mode = dnf5_mode)
+                             dnf5_mode = dnf5_mode,
+                             dnfdaemon_mode = dnfdaemon_mode)
 
     write_config(context)
 
+    if dnfdaemon_mode == True:
+        p_dbus_daemon = subprocess.Popen(['/usr/bin/dbus-daemon', '--system'])
+        p_polkitd = subprocess.Popen(['/usr/lib/polkit-1/polkitd', '&'])
```

Comment:
We should either terminate the processes after the scenario or detect that they are already running and do not try to launch them again. Otherwise I would expect strange things happening once more than one scenario is tagged with `@dnfdaemon`